integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/wait/service/WaitService.java

```diff
@@ -6,14 +6,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.sequenceiq.cloudbreak.polling.AttemptBasedTimeoutChecker;
+import com.sequenceiq.cloudbreak.polling.TimeoutChecker;
 import com.sequenceiq.it.cloudbreak.context.TestContext;
 
 public class WaitService<T extends WaitObject> {
     private static final Logger LOGGER = LoggerFactory.getLogger(WaitService.class);
 
     public Result<WaitResult, Exception> waitObject(StatusChecker<T> statusChecker, T t, TestContext testContext, Duration interval, int maxAttempts,
             int maxRetryCount) {
-        return waitObject(statusChecker, t, testContext, interval, new TimeoutChecker(maxAttempts), maxRetryCount);
+        return waitObject(statusChecker, t, testContext, interval, new AttemptBasedTimeoutChecker(maxAttempts), maxRetryCount);
```

Comment:
Is it necessary to change the default time checker?

Reply:
it's not changed, it's renamed. `TimeoutChecker` is an interface now, etc